packages/connectors/connector-smsaero/src/mock.ts

```diff
@@ -0,0 +1,17 @@
+import type { SmsAeroConfig } from './types.js';
+
+const mockedEmail = 'test@test.com';
+const mockedApiKey = 'api-key';
+const mockedSenderName = 'sender-name';
+
+export const mockedConfig: SmsAeroConfig = {
+  email: mockedEmail,
+  apiKey: mockedApiKey,
+  senderName: mockedSenderName,
+  templates: [
+    {
+      usageType: 'Test',
+      content: 'This is for testing purposes only. Your verification code is {{code}}.',
+    },
+  ],
```

Comment:
nit-picking: This is out-of-date, as we don't have `Test` usageType in templates any more.